ReactAndroid/src/main/java/com/facebook/react/modules/systeminfo/AndroidInfoHelpers.java

```diff
@@ -12,8 +12,8 @@
   public static final String GENYMOTION_LOCALHOST = "10.0.3.2";
   public static final String DEVICE_LOCALHOST = "localhost";
 
-  private static final int DEBUG_SERVER_HOST_PORT = 8081;
-  private static final int INSPECTOR_PROXY_PORT = 8082;
+  public static final int DEBUG_SERVER_HOST_PORT = 8081;
+  public static final int INSPECTOR_PROXY_PORT = 8082;
```

Comment:
There was no compelling reason to keep these values private. My WebWorker library checks for conflicts against these defaults (and utilizes the `getServerIpAddress` below to avoid duplicating logic to find the loopback interface).